src/playlist-selectors.js

```diff
@@ -166,8 +169,16 @@ export const simpleSelector = function(
     height: playerHeight,
     limitRenditionByPlayerDimensions
   };
+
+  let playlists = master.playlists;
+
+  // if playlist is audio only, select between currently active audio group playlists.
+  if (Playlist.isAudioOnly(master)) {
+    playlists = masterPlaylistController.getAudioTrackPlaylists_();
+    options.audioOnly = true;
```

Comment:
Is this used anywhere?

Reply:
We print `options` to the debug log. I think it would be helpful to know that we are selecting an audio only playlist.